src/mono/wasm/runtime/rollup.config.js

```diff
@@ -216,3 +238,21 @@ function regexReplace(replacements = []) {
         return { code: fixed };
     }
 }
+
+// Finds all files that look like a webworker toplevel input file in the given path.
+// Does not look recursively in subdirectories.
+// Returns an array of objects {"workerName": "foo", "path": "/path/dotnet-foo-worker.ts"}
+//
+// A file looks like a webworker toplevel input if it's `dotnet-{name}-worker.ts` or `.js`
+function findWebWorkerInputs(basePath) {
```

Comment:
Is this slow ? Could we tell OS what is the filter instead ?
We have [fast-glob 3.2.7](https://www.npmjs.com/package/fast-glob) already in dependencies, so you could add it explicitly and use it instead of this.

`const fg = require('fast-glob');`

Reply:
Updated to use fast-glob, thanks

Reply:
> Yesterday I thought we are searching the whole /src and now I noticed that the file was moved to wasm/runtime which is even better. Otherwise using ignore option would be good to avoid various bin folders.

It's only looking in `src/mono/wasm/runtime/workers`, and we're not looking at subdirectories at all